client/src/app/tabs/cloud-bpmn/modeler/BpmnModeler.js

```diff
@@ -18,6 +18,11 @@ import handToolOnSpaceModule from '../../bpmn/modeler/features/hand-tool-on-spac
 import propertiesPanelKeyboardBindingsModule from '../../bpmn/modeler/features/properties-panel-keyboard-bindings';
 import lintingAnnotationsModule from '@camunda/linting/modeler';
 
+import {
+  BpmnJSTracking as bpmnJSTracking,
+  BpmnJSTrackingModules as bpmnJSTrackingModules
```

Comment:
My favorite would actually be to explicitly expose different tracking modules here, and include them.

This way it is clear "what we use". But let's get this integrated for now and consider moving to explicit tracking features in the future.

Reply:
I think both would work. If we go the import `lib` path I'd probably rename it from `trackingModules` to `tracking` or even better `features`.

So `'bpmn-js-tracking/lib/trackingModules/popupMenu'` becomes `'bpmn-js-tracking/lib/features/popup-menu'`;